arch/arm64/boot/dts/qcom/sdm660.dtsi

```diff
@@ -0,0 +1,375 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * Copyright (c) 2018, Craig Tatlor.
+ * Copyright (c) 2020, The Linux Foundation. All rights reserved.
```

Comment:
This line was copied from opendata's original MR, (he probably copied it from downstream), but I changed date to here 2020 too, because I thought that's kind of "release date"..

Reply:
IANAL, I would keep the original date.

Reply:
:ok: